daml-assistant/src/DAML/Assistant/Env.hs

```diff
@@ -104,7 +105,11 @@ getSdk damlPath projectPathM =
     wrapErr "Determining SDK version and path." $ do
 
         sdkVersion <- firstJustM id
-            [ fmap (SdkVersion . pack) <$> lookupEnv sdkVersionEnvVar
+            [ lookupEnv sdkVersionEnvVar >>= \ vstrM -> pure $ do
+                vstr <- vstrM
+                v <- eitherToMaybe (V.fromText (pack vstr))
+                pure (SdkVersion v)
```

Comment:
```suggestion
                Just (SdkVersion v)
```
Given that there is another use of `pure` 3 lines above this for a different instance, I think it’s easier to understand this if you make it explicit which instance is being used.

Reply:
yep, this makes sense